third_party/move/move-compiler-v2/tests/bytecode-generator/bug_14471_receiver_inference.move

```diff
@@ -0,0 +1,28 @@
+module 0x815::m {
```

Comment:
Please also add the test from https://github.com/aptos-labs/aptos-core/issues/14030 to demonstrate it is closed as well.

Reply:
Not easily possible since it uses framework functionality, but I engineered this test to cover the pattern of both, and rather sure both bugs are fixed.